odoo/tools/translate.py

```diff
@@ -1026,7 +1027,11 @@ def _export_imdinfo(self, model: str, imd_per_id: dict[int, ImdInfo]):
                 # From our business perspective, the parent column is no need to be translated,
                 # but it is need to be set to jsonb column, since the child columns need to be translated
                 # And export the parent field may make one value to be translated twice in transifex
-                if not field.translate or not field.store or str(field) == 'ir.actions.actions.name':
+                #
+                # Some ir_model_fields.field_description are filtered
+                # because their fields have falsy attribute string_translate
+                if not field.translate or not field.store or str(field) == 'ir.actions.actions.name' \
+                        or (str(field) == 'ir.model.fields.field_description' and not env[record.model]._fields[record.name].string_translate):
```

Comment:
Please make this humanly parseable :wink: 
```suggestion
                if (
                    not (field.translate and field.store)
                    or str(field) == 'ir.actions.actions.name'
                    or (str(field) == 'ir.model.fields.field_description'
                        and not env[record.model]._fields[record.name].string_translate)
                ):
```
I wonder whether the last condition could fail, in case the target field is not in the registry, but could this happen? :thinking:

Reply:
(https://github.com/odoo/odoo/blob/eab9453a0dff21fb3672a05eed346e073563a5a6/odoo/tools/translate.py#L1086)

I think the invalid fields has already been removed here